src/plugin.ts

```diff
@@ -1,12 +1,19 @@
 import { Context } from "./types/context";
 import { userStartStop } from "./handlers/user-start-stop";
 import { Env, PluginInputs } from "./types";
-
 import { Octokit } from "@octokit/rest";
 import { createClient } from "@supabase/supabase-js";
 import { createAdapters } from "./adapters";
 import { Logs } from "@ubiquity-dao/ubiquibot-logger";
 
+export type UserRole = "admin" | "member" | "contributor";
```

Comment:
This would probably be a dynamic union based on the keys of the passed in settings or a type which contains the pre-defined roles and can be extended